iroha/src/queue.rs

```diff
@@ -61,72 +154,60 @@ impl Queue {
         }
     }
 
-    /// Gets at most `maximum_transactions_in_block` number of transaction, but does not drop them out of the queue.
-    /// Drops only the transactions that have reached their TTL or are already in blockchain.
-    /// For MST transactions if on leader, waits for them to gather enough signatures before, showing them as output of this function.
+    /// Gets at most `max_txs_in_block` number of tx, but does not drop them out of the queue.
+    /// Drops only the txs that have reached their TTL or are already in blockchain.
+    /// For MST txs if on leader, waits for them to gather enough signatures before, showing them as output of this function.
```

Comment:
In docs better to use `transaction` when it is not citation

Reply:
Also this is kinda weird comment. Couldn't get my head around it and around fsm which is there. I think it should be substituted by some simple datastructure which does some ttl management.

Reply:
But that i guess can be made in other pr after discussion of sumeragi design.